### ISSUE TITLE
Repeated dialogue

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#317]
```
What steps will reproduce the problem?
1. Talk to an NPC and stay on conversation screen.
2. Turn phone/tablet screen off or wait for it to go off on its own.
3. Turn the screen back on and the conversation as well as any toast is 
repeated.

Please use labels and text to provide additional information.
More information available here: 
http://andorstrail.com/viewtopic.php?f=3&t=1991&p=22155
```

Original issue reported on code.google.com by `sdeva...@gmail.com` on 27 Aug 2012 at 4:43

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 05b2f7ee049a4974a3dec282a8268a628c54da2f. Thanks nether!
```

Original comment by `oskar.wi...@gmail.com` on 13 Sep 2012 at 5:08
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:44
- Changed state: **Fixed**
